kubernetes/samples/scripts/common/utility.sh

```diff
@@ -926,3 +926,27 @@ function checkService(){
  done
  echo "Service [$svc] found"
 }
+
+# Get pod name when pod available in a given namespace
+function getPodName(){
+
+ max=120
+ count=1
+
+ pod=$1
+ ns=$2
+
+ pname=`kubectl get po -n ${ns} | grep -w ${pod} | awk '{print $1}'`
+ while [ -z ${pname} -a $count -le $max ] ; do
```

Comment:
I think ${pname} must be encased in quotes. This will not parse correctly if pname is empty.